ivy/functional/frontends/torch/indexing_slicing_joining_mutating_ops.py

```diff
@@ -215,18 +215,40 @@ def split(tensor, split_size_or_sections, dim=0):
     )
 
 
+def _get_indices_or_sections(indices_or_sections, indices, sections):
+    if not indices_or_sections:
+        if isinstance(indices_or_sections, (list, tuple)):

```

Comment:
this condition is a bit difficult to read/understand, maybe extending the first condition to be `if not indices_or_sections and not isinstance(indices_or_sections, (list, tuple)):` is slightly more readable,  a comment wouldn't hurt either if you see fit:D! (e.g. indices_or_sections can accept empty lists or tuples)